INSTALL.md

```diff
@@ -1,23 +1,113 @@
 # Contents
-- [Contents](#contents)
-  - [Basic Configuration](#basic-configuration)
-  - [Adding Applications](#adding-applications)
-  - [Adding Kratos to Path](#adding-kratos-to-path)
-  - [Examples](#examples)
-    - [Linux](#linux)
-    - [Windows](#windows)
-    - [MacOS](#macos)
-  - [Advanced Configuration](#advanced-configuration)
-    - [Building Environment](#building-environment)
-    - [Common Flags](#common-flags)
-    - [Compilation Performance](#compilation-performance)
-    - [Parallelism](#parallelism)
-    - [Logging](#logging)
-    - [External libraries](#external-libraries)
-      - [Metis](#metis)
-      - [Trilinos](#trilinos)
-  - [Applications](#applications)
+* [Cloning Kratos](#cloning-kratos)
+* [Kratos Dependencies](#kratos-dependencies)
+  * [Kratos Core Dependencies](#kratos-core-dependencies)
+    * [Linux Installation](#linux-installation)
+    * [Windows Installation](#windows-installation)
+  * [Specific Application Dependencies](#specific-application-dependencies)
+* [Basic Configuration](#basic-configuration)
+* [Adding Applications](#adding-applications)
+* [Adding Kratos to Path](#adding-kratos-to-path)
+* [Examples](#examples)
+  * [Linux](#linux)
+  * [Windows](#windows)
+  * [MacOS](#macos)
+* [Advanced Configuration](#advanced-configuration)
+  * [Building Environment](#building-environments)
+  * [Common Flags](#common-flags)
+  * [Compilation Performance](#compilation-performance)
+  * [MPI-Parallelism](#parallelism)
+  * [External Libraries](#external-libraries)
+    * [Metis](#metis)
+    * [Trilinos](#trilinos)
+
+## Cloning Kratos
+
+In order to obtain the source code of Kratos, you will need to clone the repository using git.
+
+You can install git through the following command in Linux:
+```Shell
+sudo apt-get install git
+```
+In Windows, you can download it in:
+
+* [Download Git](https://git-scm.com/downloads)
+
+
+
+Once git is installed you can fetch the code by using this command in a terminal:
+
+```Shell
+git clone https://github.com/KratosMultiphysics/Kratos Kratos
+```
+
+## Kratos Dependencies
+
+### Kratos Core Dependencies
+  These are the basic dependecies needed to compile the Kratos Core and most of the Kratos applications.
+  * Python3-dev
+  * C++11 compiler
+  * CMake
+  * Boost (dependencies are header-only, no compilation of boost libraries required)
+
+Additionaly Visual Studio or MinGW is required to compile in Windows.
```

Comment:
@philbucher help here. Are there still solvers that require the external by default? If not we can remove this from the wiki 8finally)

Reply:
you mean ExternalSolversApplication

not really, the major Apps have bee refactored, I would remove it from the readme

Reply:
Perfect. then we can also remove everything related to MinGW and fotran libs for win :D